README.md

```
@@ -17,6 +17,61 @@ Corresponding [EGRA skill](https://github.com/elimu-ai/model/blob/master/src/mai
+./gradlew assembleBoardDebug
+```
+
+### Install on device
+
+```bash
+./gradlew installFurguardiansDebug
+```
+
+### APK output
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)